packages/components/src/core/NavigationHeader/components/NavigationHeaderPrimaryNav/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { SdsTagColorType } from "src/core/Tag";
+import { PrimaryNavItem, StyledLabel } from "./style";
+import { ReactNode } from "react";
+import { StyledTag } from "../../style";
+import { StyledSection } from "../style";
+
+export interface NavigationHeaderPrimaryNavItem<T extends string> {
+  key: T;
+  label: ReactNode;
+  tag?: string;
+  tagColor?: SdsTagColorType;
+}
+
+export interface NavigationHeaderPrimaryNavProps<T extends string> {
+  hasInvertedStyle?: boolean;
+  items: NavigationHeaderPrimaryNavItem<T>[];
+  onChange(value: T): void;
+  value: T;
+}
+
+export default function NavigationHeaderPrimaryNav<T extends string>({
```

Comment:
This could be VCP's design not adhering to SDS recommendations, but VCP actually has a primary navigation with dropdown. Should that be a supported use case?

<img width="1227" alt="Screenshot 2024-12-05 at 2 22 07 PM" src="https://github.com/user-attachments/assets/c3d765a9-0c18-4ce1-9d13-52edb56156e2">


CC: @clarsen-czi 🙏

Reply:
Got it! Yeah only if design sees primary nav dropdown as a common pattern instead of a one time snowflake 😁 👌

Reply:
I believe Kevin's intention was that primary navigation would not include dropdowns and that dropdowns are meant to only be available on secondary nav items. It seems to me that, in the example screenshot above, the Tutorials section actually makes more sense as secondary navigation since it links to training/how-to content as opposed to core functional area of application. @kev-zunshiwang let us know what you think!

Reply:
Ahh I see! So more like VCP not following design best practices 💡 That makes sense to me 😁 Thank you!